src/controls/TextControl.tsx

```diff
@@ -64,10 +67,10 @@ export function TextControl({
       label={label}
       id={id}
       name={path}
-      rules={rules}
       validateTrigger={["onBlur"]}
-      tooltip={tooltip}
-      {...formItemProps}
+      rules={rules}
+      tooltip={mergedTooltip}
+      {...formItemPropsWOTooltip}
```

Comment:
I'm not sure I'm reading this right, but it looks like this code change isn't necessary--does the test you added pass without it?

Reply:
The test will work, but there is some additional ambiguity that needs to be addressed. This is the reason why I dislike having two tooltips. 
If someone passes both tooltips or if we need to handle tooltip properties differently, the `formItem` tooltip will take precedence. Whatever we pass to the tooltip property will be overwritten by the `FormItem` tooltip directly from the ui-schema.

Reply:
Makes sense--maybe add a test that fails without this change, then?